<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PTTCL9E" failures="0" tests="1" name="Default test" time="0.079" errors="0" timestamp="2023-10-02T19:12:39 IST">
  <testcase classname="BaseUtility.BaseClass" name="@BeforeTest lounching" time="0.012">
    <failure type="java.io.FileNotFoundException" message="C:\Users\webca\eclipse-workspace\seleniumpractice-sept2023\PropertyFile\Commondata.properties (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\webca\eclipse-workspace\seleniumpractice-sept2023\PropertyFile\Commondata.properties (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at ReadProperties.Readpro.<init>(Readpro.java:17)
at BaseUtility.BaseClass.lounching(BaseClass.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest lounching -->
  <testcase classname="BaseUtility.BaseClass" name="@BeforeMethod login" time="-1.696254159378E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod login -->
  <testcase classname="BaseUtility.BaseClass" name="@AfterTest tearDown" time="-1.69625415939E9">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase classname="Demo_Sep_TestCases.demo_Sep23_Homepage" name="validateHomepageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- validateHomepageTitle -->
</testsuite> <!-- Default test -->
